spec/unit/block_score/person_spec.rb

```diff
@@ -29,22 +31,23 @@
     it { is_expected.to be_persisted }
     its(:name_first) { is_expected.not_to be_empty }
     it { is_expected.to be_an_instance_of described_class }
-    context 'null id presented' do
-      let(:person_id) { nil }
 
-      it { expect { described_class.retrieve(person_id) }.to raise_error(ArgumentError, 'ID must be supplied') }
-    end
-
-    context 'empty id presented' do
-      let(:person_id) { '' }
-
-      it { expect { described_class.retrieve(person_id) }.to raise_error(ArgumentError, 'ID must be supplied') }
+    context 'no id presented' do
+      it 'has to have some ID' do
+        expect { described_class.retrieve(nil) }
+          .to raise_error(ArgumentError, 'ID must be supplied')
+        expect { described_class.retrieve('') }
+          .to raise_error(ArgumentError, 'ID must be supplied')
+      end
     end
 
     context 'Malformed ID presented' do
       let(:person_id) { 'ABC&&234' }
 
-      it { expect { described_class.retrieve(person_id) }.to raise_error(ArgumentError, 'ID is malformed') }
+      it 'has to have a properly formed ID' do
```

Comment:
We are only really testing to a limited degree it has the right characters - the backend API will tell us if it is truly `valid`? I am concerned about the mixup on the term `valid` because persons are validated by the API. So reading the tests could be confusing.